tests/ts_aggregation_simple.erl

```diff
@@ -77,7 +84,9 @@ verify_aggregation(ClusterType) ->
        <<"COUNT(temperature)">>,
        <<"COUNT(precipitation)">>
       ],
-      [{Count, Count, Count}]},
+      [{count_non_nulls(?PRESSURE_COL_INDEX, Data),
```

Comment:
@andytill I thought this test had previously passed.  It is simply because not an entire row was NULL?